.gitmodules

```diff
@@ -7,3 +7,7 @@
 [submodule "qemu_mode/qemuafl"]
 	path = qemu_mode/qemuafl
 	url = https://github.com/AFLplusplus/qemuafl
+[submodule "frida_mode/frida"]
+	path = frida_mode/frida
+	url = https://github.com/WorksButNotTested/frida.git
```

Comment:
Shouldn't we depend on upstream frida istead?

Reply:
current upstream frida has a breaking bug. this is not for release but so we can push to stable for the moment.